pages/index.vue

```diff
@@ -1,42 +1,53 @@
 <template>
 
-	<b-container>
+	<b-container fluid>
 
-		<h1>Origami Annotation Tool</h1>
+		<!-- Navigation bar -->
+		<tool-navbar :navItemsState="navItemsState" :pageName="fullName"></tool-navbar>
 
-		<!-- Debug component - shows file contents -->
+		<!-- TSV file loading area -->
 		<b-row>
 			<h2>TSV File</h2>
-			<textarea rows="5" cols="200" v-model="stringifiedTsvFile"></textarea>
-		</b-row>		
 
-		<!-- Selects participant.tsv file -->
-		<file-selector 
-			content-type="text/tab-separated-values"
-			v-on:file-selected="saveTsvFileData($event)">
-		</file-selector>
+			<!-- Debug component - shows file contents -->			
+			<textarea rows="5" cols="200" v-model="stringifiedTsvFile" :key="currentState"></textarea>
+			
+			<!-- Selects participant.tsv file -->
+			<file-selector 
```

Comment:
JS curiosity question: what's the benefit of this over something like [`<b-form-file>`](https://bootstrap-vue.org/docs/components/form-file)?

Reply:
The chance to be more explicit about a component's functionality. (Of course the 'b' in 'b-form-file' references Bootstrap.) But the more important point here is that defining our own components, even if they're virtually a wrapper for an already written third party component, allows us to group with it custom functionality for our app.

Is that what you meant?